### PR TITLE
Dashboard misc

### PR DESCRIPTION
**Pull request by intrepiditee**
Misc files such as the entrypoint and configurations.

### CODE REVIEW

File: import-automation/progress-dashboard-rest/app.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+runtime: python37
+entrypoint: gunicorn -b :$PORT app.main:FLASK_APP
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app.yaml:2]: How did we choose gunicorn?  What services does it provide for us?

File: import-automation/progress-dashboard-rest/app/main.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Entry point of the API.
+When deployed on App Engine, gunicorn will serve the FLASK_APP variable,
+which is a Flask object.
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app/main.py:18]: How does Flask fit into the services we are using?  Do we have a diagram?

File: import-automation/progress-dashboard-rest/app/main.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+from app import utils
+
+
+def create_app(logging=True):
+    """Creates the Flask app.
+
+    Args:
+        logging: Whether to set up Google Cloud Logging.
+    """
+    if not logging:
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app/main.py:36]: Is the choice between Google Cloud Logging and Python logging?  Xor is this logic accidentally reversed?

File: import-automation/progress-dashboard-rest/.gitignore [hunk 4]
```diff
@@ -0,0 +1,141 @@
+.DS_Store
```
[the remark this answers is not in the document]
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/.gitignore:1]: Files and directories that match any pattern in .gitignore will not be added to a commit

File: import-automation/progress-dashboard-rest/app/main.py (hunk 2 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/main.py:18]: Flask is just a web API framework

File: import-automation/progress-dashboard-rest/app/main.py (hunk 3 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/main.py:36]: Ops. It is reversed.

File: import-automation/progress-dashboard-rest/app.yaml (hunk 1 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app.yaml:2]: gunicorn is an application server that serves the flask app. This https://serverfault.com/questions/331256/why-do-i-need-nginx-and-something-like-gunicorn may be helpful.

File: import-automation/progress-dashboard-rest/app/main.py (hunk 3 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/main.py:36]: fixed

**Pull request merged**